### PR TITLE
Add "Privacidade" page

### LINKED ISSUE #48: [frontend][landing] Create a "Privacidade" page
Create this page with the copy from the google doc:
Covidografia-PrivacidadeDados

### PR DESCRIPTION
**Pull request by diogoteix**
Closes #48

### CODE REVIEW

**Review by fabioscsilva — APPROVED** [review]

File: src/sections/privacidade/index.jsx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            correlacionáveis com dados de contacto. Isto foi possível graças à
+            forma especial como os dados são tratados e armazenados. Veja mais
+            em{" "}
+            <Typography
+              variant="smallBody"
+              weight="bold"
+              color="purple"
+              className={styles.inlineText}
+            >
+              Como tratamos os seus dados.
```
Comment (fabioscsilva) [inline src/sections/privacidade/index.jsx:35]: This should be an anchor to "Como tratamos os seus dados" section, right?

File: src/sections/privacidade/index.jsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+            Geral de Proteção de Dados pessoais (RGPD), desde o desenho dos
+            processos com princípios fundamentais como “Privacy by Design” e
+            “Data Minimization” (ver{" "}
+            <Typography
+              variant="smallBody"
+              weight="bold"
+              color="purple"
+              className={styles.inlineText}
+            >
+              Como tratamos os seus dados.
```
Comment (fabioscsilva) [inline src/sections/privacidade/index.jsx:59]: The same here

**Comment by ruisaraiva19** [comment]
@diogoteix please delete the `package-lock.json` file. We are using Yarn on this project 🙂.

**Review by ruisaraiva19 — CHANGES_REQUESTED** [review]
@diogoteix please delete the `package-lock.json` file. We are using Yarn on this project 🙂.

File: src/sections/privacidade/index.jsx (hunk 1 above)
Comment (diogoteix) [reply src/sections/privacidade/index.jsx:35]: Done!

File: src/sections/privacidade/index.jsx (hunk 2 above)
Comment (diogoteix) [reply src/sections/privacidade/index.jsx:59]: Done!

**Comment by diogoteix** [comment]
@ruisaraiva19 Done!

**Review by nunofgm — APPROVED** [review]

**Comment by fabioscsilva** [comment]
@ruisaraiva19 The change that you requested is already addressed, can you unlock the PR?

**Review by ruisaraiva19 — APPROVED** [review]

**Pull request merged**